index.html

```diff
@@ -45,6 +45,7 @@
 
     <div id="configHeader"><b>Advanced settings:</b> [<a id="configToggleBtn" href="#">show</a>]</div>
     <div id="config">
+        <input id="sound_toggle" name="sound_toggle" type="checkbox" checked> <label for="sound_toggle">Enable Sound</label><br/>
```

Comment:
Can you please add this as the last setting, instead of the first? Seed, num outputs, etc are used more frequently.

Also, how about naming this "Play sound on completing the task"? Probably clearer to people what the sound option is for. Thanks!